filters/ThirdParty/filter_249_NorwegianList/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "filterId": 249,
+  "name": "Dandelion Sprout's Norwegian Filters",
+  "description": " Dandelion Sprout's Norwegian Filters for tidier websites.",
+  "timeAdded": 1522412113779,
+  "homepage": "https://github.com/DandelionSprout/adfilt",
+  "expires": "2 days",
+  "displayNumber": 249,
+  "groupId": 3,
```

Comment:
Wrong group, needs to be 7 -- language-specific

Reply:
Done